config.ini.example

```diff
@@ -85,7 +85,9 @@ url = https://fahrplan.events.ccc.de/congress/YEAR/Fahrplan/schedule.json
 
 [plugins.dates]
 url = http://www.google.com/calendar/ical/2eskb61g20prl65k2qd01uktis%40group.calendar.google.com/public/basic.ics
+cache = /tmp/dates.cache
 # timedelta = 21
+# announce_minutes = 60,1440
```

Comment:
´´´íni
# Minutes in the future to show event announcement (list of multiple values)
announce_minutes[] = 60
announce_minutes[] = 1440´´´

results in

configparser.DuplicateOptionError: While reading from 'config.ini' [line 99]: option 'announce_minutes[]' in section 'plugins.dates' already exists